### PR TITLE
Profile3

### PR DESCRIPTION
**Pull request by hrafov**

### CODE REVIEW

**Review by KnazevValeriy — DISMISSED** [review]

**Review by MysiukR — DISMISSED** [review]

File: src/main/java/com/epam/lab/business/ProfilePageBo.java [hunk 1]
```diff
@@ -133,4 +133,54 @@ public void verifyWallPage() {
     public String getFullClassname(String firstPart, String secondPart) {
         return String.format("%s%s", firstPart, secondPart);
     }
+    
+    @Step("PageElementBO Step:  get and click All Contacts Button...")
+	     public void getAndClickAllContactsButton() {
```
Comment (old-zoomer) [inline src/main/java/com/epam/lab/business/ProfilePageBo.java]: the same as in previous pull request

**Review by old-zoomer — CHANGES_REQUESTED** [review]

File: src/main/java/com/epam/lab/business/ProfilePageBo.java (hunk 1 above)
Comment (hrafov) [reply src/main/java/com/epam/lab/business/ProfilePageBo.java]: yes it my fault

**Review by hrafov — COMMENTED** [review]
delete 2 test

**Review by old-zoomer — APPROVED** [review]

**Review by MysiukR — APPROVED** [review]

**Review by hrafov — COMMENTED** [review]
new

**Pull request merged**